content/id/docs/tutorials/kubernetes-basics/expose/expose-intro.html

```diff
@@ -0,0 +1,109 @@
+---
+title: Menggunakan Service untuk Ekpos Aplikasimu
+weight: 10
+---
+
+<!DOCTYPE html>
+
+<html lang="id">
+
+<body>
+
+<link href="/docs/tutorials/kubernetes-basics/public/css/styles.css" rel="stylesheet">
+
+<div class="layout" id="top">
+
+	<main class="content">
+
+		<div class="row">
+			<div class="col-md-8">
+    		<h3>Tujuan</h3>
+				<ul>
+					<li>Belajar tentang Service di Kubernetes</li>
+					<li>Memahami bagaimana label dan objek LabelSelector berhubungan dengan sebuah Service</li>
+					<li>Ekpos sebuah aplikasi keluar klaster Kubernetes menggunakan sebuah Service</li>
```

Comment:
Typo and consistency!
```suggestion
					<li>Mengekspos sebuah aplikasi keluar klaster Kubernetes menggunakan sebuah Service</li>
```